style.css

```diff
@@ -38,7 +40,8 @@ main {
     min-height: 90vh;
 }
 
-h1 {
+h1,
+h2 {
```

Comment:
Added h2 elements to add styling to them. I consciously put these together to use less code, however the padding defined here `padding: 1em;` causes the accordion element to be a bit larger. I suggest we have a look at separating these elements.